src/StringUtils.h

```diff
@@ -8,7 +8,14 @@
 
 #pragma once
 
+#include <cstddef>
 
+#include <string>
+#include <list>
+#include <vector>
+#include <map>
+
+#include "fmt.h"
```

Comment:
We'd want these headers precompiled for example, @KingCol13 do you know what needed this

Reply:
AString needed <string>, then there's AStringMap AStringList and AStringVector. StringUtils.h is included in Globals.h anyway.